velox/functions/prestosql/json/tests/SIMDJsonExtractorTest.cpp

```diff
@@ -177,6 +181,17 @@ TEST_F(SIMDJsonExtractorTest, generalJsonTest) {
   testExtract(json, "$[\"e mail\"]", "\"amy@only_for_json_udf_test.net\"");
   testExtract(json, "$.owner", "\"amy\"");
 
+  // Wildcard over object's value elements
+  testExtract(
+      json,
+      "$.store.book[0].[*]",
+      std::vector<std::string>{
+          "\"Nigel Rees\"",
+          "\"ayings of the Century\"",
+          "\"reference\"",
+          "8.95"});
+  testExtract(json, "$.store.[*].price", std::vector<std::string>{"19.95"});
```

Comment:
Can you have an empty object?  E.g. {"a": {}}

Would be good to add a test for that with a wildcard (I guess that would return an empty array?)